install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -866,6 +866,12 @@ data:
   enable-bgp-control-plane: "false"
 {{- end }}
 
+{{- if .Values.pmtuDiscovery.enabled }}
+  enable-pmtu-discovery: "true"
+{{- else }}
+  enable-pmtu-discovery: "false"
+{{- end }}
```

Comment:
Nit: I know this is taken from above, but since this defaults to false, let's just remove this case.

There are similar examples below in this file.

```suggestion
```

Reply:
done